Problems/Programmers/92334/92334_ans.py

```diff
@@ -0,0 +1,17 @@
+#https://programmers.co.kr/learn/courses/30/lessons/92334?language=python3
+
+def solution(id_list, report, k):
+    report = set(report)
+    answer = {x:0  for x in id_list}
+    reported = {x:0  for x in id_list}
+    
+    for x in report:
+        reported[x.split()[1]]+=1
+    for x in report:
+        if reported[x.split()[1]]>=k:
+            answer[x.split()[0]] +=1
```

Comment:
```suggestion
    for x in report:
    	A, B = x
        reported[B]+=1
    for x in report:
    	A, B = x
        if reported[B]>=k:
            answer[A] +=1
```
가독성을 위해..나라면 이런식으로 했을 것 같음.
A,B 가 서로 다른 성격 (신고자 / 위반자) 인데 배열마냥 처리하는게 아무래도 안이쁘다 생각.